src/Okta.Idx.Sdk/IIdxClient.cs

```diff
@@ -260,5 +260,23 @@ public interface IIdxClient : IOktaClient
         /// <param name="cancellationToken">The cancellation token.</param>
         /// <returns>The password required response.</returns>
         Task<PasswordRequiredResponse> CheckIsPasswordRequiredAsync(IIdxContext idxContext, CancellationToken cancellationToken = default);
+
+        /// <summary>
+        /// Enroll a security question.
+        /// </summary>
+        /// <param name="verifyAuthenticatorOptions">The options to verify a security question.</param>
+        /// <param name="idxContext">The IDX context.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <returns>The authentication response.</returns>
+        Task<AuthenticationResponse> EnrollAuthenticatorAsync(SecurityQuestionAuthenticatorOptions verifyAuthenticatorOptions, IIdxContext idxContext, CancellationToken cancellationToken = default);
```

Comment:
Nit: Just to be consistent with other methods, can we rename SecurityQuestionAuthenticatorOptions to `EnrollSecurityQuestionAuthenticatorOptions`?